### DISCUSSION TITLE
Is it possible to support local mode?

### DISCUSSION THREAD

**Discussion by freshchen** [PrefectHQ/prefect#15980] [General]
### Describe the current behavior

Thanks to the prefect project, I can simply add @flow and @task annotations to complete the integration. At present, I only use the observable, retry, and submit mapreduce functions of prefect. In the deployment scenario, it may not be possible to deploy the prefect server. Is there a switch that can make my code run normally without errors?

Suppose I start a business process and a prefect server process using sqlite in POD at the same time, is there a way for prefect to automatically clean up the data in sqlite to prevent my POD storage from growing larger?

Now I will get this error
  File "/opt/miniconda3/envs/prefect-demo/lib/python3.11/site-packages/prefect/client/orchestration.py", line 226, in get_client
    raise ValueError(
ValueError: No Prefect API URL provided. Please set PREFECT_API_URL to the address of a running Prefect server.

### Describe the proposed behavior

As above

**Comment by cicdw** [comment]
Hey @freshchen, good question (fyi I've converted your original issue to a discussion). This is possible to configure! We call what you're describing "ephemeral mode"; you can enable this mode of execution by setting `PREFECT_SERVER_ALLOW_EPHEMERAL_MODE=true` as an environment variable.

It should behave as you've described but let me know if you run into an unexpected issue!
